tubular/nominal.py

```diff
@@ -1246,9 +1247,12 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
                     )
 
         # Drop original columns
```

Comment:
I think the comment here could do with extending with ' if self.drop_original is true' as the functionality is not obvious without reading the called function.